assets/js/feedback.js

```diff
@@ -1,109 +0,0 @@
-var tas= [];
```

Comment:
This whole file was deprecated in favour of `student.js` weeks ago. This is just cleanup.